src/app/connections/list-toolbar/list-toolbar.component.html

```diff
@@ -45,7 +45,7 @@
           </button>
       </div>
       <div class="toolbar-pf-action-right">
-        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['create']">Create
+        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['/connections/create']">Create
```

Comment:
Oh wait, I thought you meant regarding the `showCreate` var.. the routerLink with `/connections/create` seems to be working fine for me.